reactioncommerce/reaction
# [WIP] 2270 Product Grid Double Click on Mobile

Resolves #2270 

**Testing**
- Clone this branch and run on your local machine
- Log In as an Admin and toggle `Edit Mode` ON
- Reduce your browser size to less that `949px` and open a new tab and visit `localhost:3000`
- Double-click on any product
- Observe that it takes you to the Product Detail Page
